### PR TITLE
QAE-881 - E2E test case for transparent header social icon text and background color, Search icon color

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Header-builder → Transparent Header → Design → Social icon and background color, Search icon color

### Screenshots

https://share.getcloudapp.com/E0ugLgWP

### How has this been tested?

How To test ?: npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+						0: 'social-icons-1',
+					},
+				},
+			},
+		};
+		await setCustomize( searchIconColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .ast-header-search .ast-icon' );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:115]: @SimranChougule1 test case failing here. Please check this at your end

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:118]: Resolved

**Review by bsfsnehal — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+						0: 'search',
+					},
+				},
+			},
+		};
+		await setCustomize( searchIconColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .ast-header-search .ast-icon' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:118]: Could you please verify selector:- https://share.bsf.io/YEurz8pb

File: tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+						0: 'social-icons-1',
+					},
+				},
+			},
+		};
+		await setCustomize( socialIconTextColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .ast-header-social-wrap .ast-social-color-type-custom .ast-builder-social-element svg' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:33]: Could you please verify selector for tex colors:- https://share.bsf.io/bLuKyJwL

File: tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+						0: 'social-icons-1',
+					},
+				},
+			},
+		};
+		await setCustomize( socialIconBgColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .ast-header-social-wrap .ast-social-color-type-custom .ast-builder-social-element' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:81]: Could you please verify selectors:- https://share.bsf.io/Bluo2pYE

File: tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js (hunk 3 above)
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:33]: Add text option and their text selector:- https://share.bsf.io/p9u6Yj2L

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/social-icon-text-bg-color-search-color.test.js:34]: added label test case also

**Pull request merged**